src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -15,6 +16,10 @@ const PAGINATION_MENU =
   'div[data-ouia-component-type="PF4/PaginationOptionsMenu"]';
 const TOOLBAR_CONTENT = '.pf-c-toolbar__content';
 const TOOLBAR = '.pf-c-toolbar';
+const DEFAULT_ROW_COUNT = 20;
+// FIXME is this shared by all tables?
```

Comment:
The pagination values are implicitly given by the `PrimaryToolbar` component that we import from `@redhat-cloud-services/frontend-components` package and the `Pagination` component that we import from `@patternfly/react-core`. We do not override them in any of the tables, which means yes, the values are shared across all tables. I think this is OK to leave it here, in spec, for now.

Reply:
Unfortunately, it is not possible because they do not have an `export` statement for the Pagination values https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/Pagination/Pagination.js#L10. Thus, you can't import and share this object. Adding `export`, however, must allow doing this. Do you want me to create a patch?

Reply:
OK, I see. Thanks for the clarification. Then, given than all tables (with pagination) will be using the same values, is there a way in our repo to put `const PAGINATION_VALUES = [10, 20, 50, 100];` that we can then import from our cypress test files? Just to avoid copying that in all cypress test files.

Reply:
yep, you can simply define the array and export it. this is how it goes if you have two modules (js files) A and B:
A.js: 
```
...
export const PAGINATION_VALUES = [10, 20, 50, 100];
...
```
B.js:
```
import { PAGINATION_VALUES } from './A.js';
...
(use PAGINATION_VALUES here)
```